source/conf.py

```diff
@@ -27,13 +27,15 @@
 versions = {
     "stable": {
         "rapids_version": stable_version,
+        "rapids_api_docs_version": stable_version,
```

Comment:
@jacobtomlinson does this templating make sense?

Right now these docs have a mix of hard-coded `/stable` or `/nightly` in their `docs.rapids.ai/api` links. For example:

https://github.com/rapidsai/deployment/blob/84c6375469d93c4e59229ba881d6911dbcba510e/source/platforms/kubeflow.md?plain=1#L70

https://github.com/rapidsai/deployment/blob/84c6375469d93c4e59229ba881d6911dbcba510e/source/platforms/databricks.md#L107

I was thinking that that could mean that if you're looking at the 24.04 deployment docs, you'd possibly see links to the `/stable` docs that take you to, say, `cuml`'s 24.08 site.

But now that I look around... is it even possible to look at the deployment docs for old releases?

Or do we only publish to these two places?

```text
https://docs.rapids.ai/deployment/stable/
https://docs.rapids.ai/deployment/nightly
```